components/Location.js

```diff
@@ -14,6 +15,13 @@ class Location extends React.Component {
     showTitle: true,
   };
 
+  isGoogleMapsAvailable() {
+    if (typeof window.google === 'object' && window.google.maps.places.AutocompleteService) {
```

Comment:
What if `window.google.maps` is set but `window.google.maps.places` undefined?

This is why I previously proposed using lodash and `window && get(window, 'google.maps.places.AutocompleteService')`.

Reply:
If `window.google.maps.places` is undefined the function will return false, which means that `window.google.maps.places.AutocompleteService` is not available.

I tried to use `window && get(window, 'google.maps.places.AutocompleteService')` but I got the error "get is not defined".

Reply:
Cool. Get is from `lodash`, you want to import it first.

`import { get } from 'lodash';`